PrebidMobile/ConfigurationAndTargeting/UserConsentDataManager.swift

```diff
@@ -61,6 +65,16 @@ public class UserConsentDataManager: NSObject {
         UserDefaults.standard.getObjectFromUserDefaults(forKey: IABUSPrivacy_StringKey)
     }
     
+    // MARK: - GPP
+    
+    public var gppHDRString: String? {
```

Comment:
General question about `UserConsentDataManager` visibility. Can we make this class internal for the SDK and not visible for SDK users?

Reply:
Sure, we can make it internal